actions/access_keys/create.go

```diff
@@ -15,7 +15,7 @@ import (
 // @Description	Create access key
 // @Tags		Access-key
 // @Produce		json
-// @Param		metadata query string false "metadata"
+// @Param		CreateAccessKey body CreateAccessKey true "CreateAccessKey model containing metadata"
```

Comment:
Maybe we should remove the description on all create request bodies.
Because I don't have any good description for that.
```suggestion
// @Param		CreateAccessKey body CreateAccessKey true ""
```

Reply:
Done